### ISSUE TITLE
Vytvořit číselník Typy tříděných odpadů

### ISSUE THREAD

**Issue by dvoriik** [opendata-mvcr/ciselniky#6]

**Comment by jakubklimek** [comment]
@dvoriik Tady chybí popis - kde se data vezmou atd.

**Comment by dvoriik** [comment]
@dvoriik ano, dával sem to do trella viz https://trello.com/c/BuiLWkyu/86-vytvořit-číselník-typy-tříděných-odpadů ... Jde o to, že číselník typů odpadů dle vyhlášky https://www.katalogodpadu.cz/Katalog-odpadu.pdf je příliš rozsáhlý a nejsou tam položky, které my potřebujeme více agregované (např. plast). Proto p. Kubásek přišel s vydefinovaným seznamem, který se používá ve tříděných odpadech. Není tak rozsáhlý jako katalog odpadů a zase není tak osekaný, jako např. jen plast, kov, textil, bioodpad apod. V EU vocabularies jsme něco jako typy tříděných odpadů nenašli, stejně tak na wikidatech je ten sorted waste poměrně osekaný.

**Comment by jakubklimek** [comment]
@dvoriik No, pokud se jedná o strukturování toho čísleníku, tak on nemusí být placatý. Naopak. může to být hierarchie. Nešlo by tedy použít ten úplný a jen pro jednoduchost přidat agragační úroveň s tím, že zachováme mapování co patří do které agregace? Když vymyslíme samostatný a nepropojený číselník, tak to bude škoda... si myslim.
